src/server/scripts/Spells/spell_rogue.cpp

```diff
@@ -640,15 +652,11 @@ class spell_rog_stealth : public SpellScriptLoader
                         if (l_Caster->HasAura(SPELL_ROGUE_SHADOW_FOCUS_AURA))
                             l_Caster->CastSpell(l_Caster, SPELL_ROGUE_SHADOW_FOCUS_COST_PCT, true);
                     }
-                    else
-                    {
-                        if (l_Caster->HasAura(eSpells::StealthSubterfuge))
-                            l_Caster->RemoveAura(eSpells::StealthSubterfuge);
```

Comment:
why do you remove the `RemoveAura` now?

Reply:
Sorry Im doing too much things in the same time...